LetsDinnerV2 MessagesExtension/View/CustomUIView/SectionSelectionInput.swift

```diff
@@ -18,7 +18,7 @@ enum DefaultSectionName {
     var labelString: String {
         switch self {
         case .miscellaneous:
-            return "Miscellaneous"
+            return LabelStrings.miscellaneous
```

Comment:
Enums are not made for single values

Reply:
That is a setup for future use, in case there is other new thing view being reused. Thanks

Reply:
Future use? Such as? Plus if that's the case why not injecting the default section name at init?